client/login/magic-login/index.jsx

```diff
@@ -932,22 +943,12 @@ class MagicLogin extends Component {
 			redirectTo: query?.redirect_to,
 			oauth2ClientId: query?.client_id,
 			gravatarFrom: query?.gravatar_from,
+			gravatarFlow: isGravatarFlow,
 		} );
-		let headerText = translate( 'Sign in with your email' );
-		let isEmailInputDisabled = isRequestingEmail;
```

Comment:
just making sure this change is intentional 😄 (not sure if we still use `isEmailInputDisabled` 🤷 )

Reply:
It has been moved to [here](https://github.com/Automattic/wp-calypso/pull/93133/files/e5cfd388e6b20e241f4b678b4e9f1330070db376#diff-53db3b94011d088eb9d347bbe08dd60fd909d858000f6f536e190ec93384bbacR936)